### ISSUE TITLE
High CPU usage on MacOS

### ISSUE THREAD

**Issue by fosron** [react-boilerplate/react-boilerplate#1511]
## Issue Type

- [x] Bug

## Description

High CPU usage on ```npm start``` without use of fsevents library. I had this running for more than a day and it got to about ~30% CPU usage on a i5 iMac. Installing fsevents helped.

## Steps to reproduce

Install React Boilerplate on MacOS.

## Solution

Need to add a documentation item about fsevents (specific about MacOS):

``` npm i --save-dev fsevents```

Maybe there's a way to auto-add it on install on MacOS.

# Versions

- React-Boilerplate: 3.4.0
- Node/NPM: v7.4.0 / 3.10.5
- OS : MacOS Sierra 10.12.2 (16C67)

**Comment by gihrig** [comment]
> Steps to reproduce
>
> Install React Boilerplate on MacOS.

As a collaborator of this project, I have multiple copies of react-boilerplate 3.4.0, current dev and several PR branches installed on OS X (10.11.6) and have never experienced this.

Please elaborate on exactly the steps you take to reproduce this issue.

**Comment by TimMeade** [comment]
We develop exclusively on OSX.  Never seen an issue with this boilerplate.  Lots of issues sometimes with others......

**Comment by KarandikarMihir** [comment]
I, personally, never have had to face this issue.

**Comment by sourcesoft** [comment]
I had the same exact issue yesterday, never had it before. Had to remove node_modules and run a clean `yarn install`. I think it happened when I merged my local with remote and there was some conflicts in yarn.lock I didn't handle well.I had the same exact issue yesterday, never had it before. Had to remove node_modules and run a clean `yarn install`. I think it happened when I merged my local with remote and there was some conflicts in yarn.lock I didn't handle well.
